Gemfile.lock

```diff
@@ -125,10 +131,15 @@ GEM
     nio4r (2.5.9)
     nokogiri (1.15.4-x86_64-darwin)
       racc (~> 1.4)
+    overcommit (0.60.0)
+      childprocess (>= 0.6.3, < 5)
+      iniparse (~> 1.4)
+      rexml (~> 3.2)
     parallel (1.23.0)
     parser (3.2.2.4)
       ast (~> 2.4.1)
       racc
+    pg (1.5.3)
```

Comment:
@GowthamK01 The `pg` gem seems to be duplicated. Can we check and revert this change?